CHANGELOG_COMPOSE.md

```diff
@@ -1,3 +1,9 @@
+#### Note: For the time being, we won't provide numbered releases for every new Jetpack Compose
+version. Check out our [snapshot builds](https://github.com/airbnb/lottie/blob/master/android-compose.md#getting-started) instead.
+
+# Next Version
```

Comment:
You can call this 1.0.0-alpha06-SNAPSHOT

Reply:
Sounds good!